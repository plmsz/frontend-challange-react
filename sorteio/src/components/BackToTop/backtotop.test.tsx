import { BackToTop } from '.'
import { render } from '@testing-library/react'

it('renders correctly', () => {
  const { asFragment } = render(<BackToTop />)
  expect(asFragment()).toMatchSnapshot('create')
})
