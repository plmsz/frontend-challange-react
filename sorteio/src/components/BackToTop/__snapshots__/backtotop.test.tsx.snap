// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly: create 1`] = `<DocumentFragment />`;
